hw2/03-count.js

```diff
@@ -1,3 +1,31 @@
-// Add your code here
+const input = document.querySelector("#userInput");
+input.addEventListener("keydown", handleKeyDown); // grab input on enter
```

Comment:
Not sure if you mean input upon entering text or upon pressing the enter button. If you meant pressing the enter button, this would be incorrect as this even searches for the press of any key. You can specify which key you want though.